packages/ListBox/test/specs/multi/ListBox.popover.spec.js

```diff
@@ -52,16 +52,17 @@ describe("ListBox.Popover", () => {
     expect(queryByTestId("popover.content")).toBeNull();
   });
 
-  it("should focus on option container as soon as the Popover is open", done => {
-    const { openSelect, getByTestId } = renderComponent();
+  it("should focus on the first option container as soon as the Popover when is multi", done => {
+    const { openSelect } = renderComponent();
 
     openSelect();
     // THIS IS A CRAPPY TEST. The ListBox wait until the popover finish animating
     // to trigger the focus, one we improved the way of the Popover communicated when the animation
     // ends we can improve this test :/
 
     setTimeout(() => {
-      expect(document.activeElement).toBe(getByTestId("popover.content"));
+      expect(document.activeElement.getAttribute("role")).toBe("option");
+      expect(document.activeElement.textContent).toBe("Venus");
```

Comment:
Why not sue `waitFor` here, instead of `setTimeout`? 
I bet `setTimeout` is not that robust.

Reply:
goo point! we had this test for so long that I forgot about it when I refactor it. Will change it 🙏